entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Wait for PostgreSQL
+while ! psql -Xq "$PGWEB_DATABASE_URL" -c "select 1" &>/dev/null
```

Comment:
So this is going to sleep forever?

Reply:
Yes. I can make a version which stops after a while.